tasks/sep/029-deputy-pause-module/.env

```
@@ -0,0 +1,6 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+OWNER_SAFE=0x837DE453AD5F21E89771e3c06239d8236c0EFd5E
+FOUNDATION_OPERATIONS_SAFE=0x837DE453AD5F21E89771e3c06239d8236c0EFd5E
+SECURITY_COUNCIL_GUARDIAN_SAFE=0x7a50f00e8D05b95F98fE38d8BeE366a7324dCf7E
+DEPUTY_PAUSE_MODULE=0x62f3972c56733aB078F0764d2414DfCaa99d574c
+PAUSE_DEPUTY=0x6A07d585eddBa8F9A4E17587F4Ea5378De1c3bAc
```

Not necessary since this is the next nonce to be executed, but for clarity when in the validation file the nonce is specify we need to add the value here. 
```suggestion
PAUSE_DEPUTY=0x6A07d585eddBa8F9A4E17587F4Ea5378De1c3bAc
SAFE_NONCE_0X837DE453AD5F21E89771E3C06239D8236C0EFD5E=1
```